### PR TITLE
Re-enabled Logstash healthcheck

### PR DESCRIPTION
**Pull request by marcus-bcl**

### CODE REVIEW

**Review by stevomcallister — CHANGES_REQUESTED** [review]
don't think this works

File: projects/person-search-index-from-delius/deploy/template/task-definition.yml [hunk 1]
```diff
@@ -5,7 +5,7 @@ containerDefinitions:
   - name: {{ .values.image.name }}
     image: ghcr.io/ministryofjustice/hmpps-probation-integration-services/{{ .values.image.name }}:{{ .Env.VERSION }}
     healthCheck:
-      command: ["CMD-SHELL", "exit 0"]
+      command: ["CMD-SHELL", "curl -sf localhost:9600"]
```
Comment (stevomcallister) [inline projects/person-search-index-from-delius/deploy/template/task-definition.yml:8]:
```suggestion
      command: ["CMD-SHELL", "curl -sSf localhost:9600"]
```

**Review by stevomcallister — CHANGES_REQUESTED** [review]

**Comment by marcus-bcl** [comment]
> don't think this works

I deployed the change via the pipeline here: https://github.com/ministryofjustice/hmpps-probation-integration-services/actions/runs/3013509248

The containers started up and reported healthy, so it should be okay to merge.

File: projects/person-search-index-from-delius/deploy/template/task-definition.yml (hunk 1 above)
Comment (marcus-bcl) [reply projects/person-search-index-from-delius/deploy/template/task-definition.yml:8]: Even if we show the error with `-S`, it'll get swallowed by ECS anyway won't it?  Happy to make the change though if you think it's required?

Comment (marcus-bcl) [reply projects/person-search-index-from-delius/deploy/template/task-definition.yml:8]: For my own reference as I had to look it up 😄
 https://explainshell.com/explain?cmd=curl+-sSf+localhost%3A9600

**Review by stevomcallister — APPROVED** [review]

**Pull request merged**